src/bundle/Resources/views/admin/role/view.html.twig

```diff
@@ -43,15 +43,13 @@
             })) }}
         </div>
     </div>
+{% endblock %}
 
-    {{ form_start(delete_form, { action: path('ezplatform.role.delete', {roleId: role.id}) }) }}
-    <section class="container my-4 px-5">
-        <div class="btn-toolbar" role="toolbar" >
-            <a href="{{ path('ezplatform.role.update', {roleId: role.id}) }}" class="btn btn-info">
-                {{ 'role.view.view.actions.edit'|trans|desc('Edit') }}
-            </a>
-            {{ form_widget(delete_form.delete, { 'attr': { 'class': 'btn btn-danger' } }) }}
-        </div>
-    </section>
-    {{ form_end(delete_form) }}
+{% block javascripts %}
+    {% javascripts
+    '@EzPlatformAdminUiBundle/Resources/public/js/scripts/admin.policy.list.js'
+    '@EzPlatformAdminUiBundle/Resources/public/js/scripts/admin.role_assignment.list.js'
```

Comment:
Is there a possibility that these files will be included separately into the page layout?
I mean, maybe we should avoid JS code duplication here and create one file with 2 selectors inside?